src/Libraries/IronPython/IronPythonUI.csproj

```diff
@@ -113,11 +113,11 @@
   <ItemGroup>
     <EmbeddedResource Include="DSIronPythonNodeImages.resx" />
     <EmbeddedResource Include="Properties\Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
Resource properties don't need to be made public, otherwise they start appearing in library view.